### PR TITLE
Data explorer/context menu initial

### PR DESCRIPTION
**Pull request by abist**
Initial PR for Data Explorer without the Menu and Command Registries

### CODE REVIEW

File: src/sql/workbench/parts/dataExplorer/common/nodeContextUtils.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+import { RawContextKey, IContextKeyService, IContextKey } from 'vs/platform/contextkey/common/contextkey';
+import { Disposable } from 'vs/base/common/lifecycle';
+import { IConnectionManagementService } from 'sql/platform/connection/common/connectionManagement';
+import { ConnectionProfile } from 'sql/platform/connection/common/connectionProfile';
+import { ICapabilitiesService } from 'sql/platform/capabilities/common/capabilitiesService';
+import { mssqlProviderName } from 'sql/platform/connection/common/constants';
+import { NodeType } from 'sql/workbench/parts/objectExplorer/common/nodeType';
+import { ExtensionNodeType } from 'sql/workbench/api/common/sqlExtHostTypes';
+
+export class NodeContextUtils extends Disposable {
```
Comment (Charles-Gagnon) [inline src/sql/workbench/parts/dataExplorer/common/nodeContextUtils.ts]: Suggest updating this name to be something like MssqlNodeContext instead

File: src/sql/workbench/parts/dataExplorer/common/nodeContextUtils.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+	static readonly canSelect = new Set([NodeType.Table, NodeType.View]);
```
Comment (Charles-Gagnon) [inline src/sql/workbench/parts/dataExplorer/common/nodeContextUtils.ts]: Make work item to split these out into a separate location that are referenced both here and in OE Actions

File: src/sql/workbench/parts/dataExplorer/electron-browser/nodeCommands.ts [hunk 3]
```diff
@@ -3,23 +3,36 @@
...
 import { IProgressService2 } from 'vs/platform/progress/common/progress';
 import { Registry } from 'vs/platform/registry/common/platform';
+import { NewNotebookAction } from 'sql/workbench/parts/notebook/notebookActions';
+import { BackupAction, RestoreAction } from 'sql/workbench/common/actions';
 
 export const DISCONNECT_COMMAND_ID = 'dataExplorer.disconnect';
 export const MANAGE_COMMAND_ID = 'dataExplorer.manage';
 export const NEW_QUERY_COMMAND_ID = 'dataExplorer.newQuery';
 export const REFRESH_COMMAND_ID = 'dataExplorer.refresh';
+export const NEW_NOTEBOOK_COMMAND_ID = 'dataExplorer.newNotebook';
```
Comment (Charles-Gagnon) [inline src/sql/workbench/parts/dataExplorer/electron-browser/nodeCommands.ts]: Clean these up - you shouldn't need them in this PR

File: src/sql/workbench/parts/dataExplorer/common/nodeContextUtils.ts (hunk 2 above)
Comment (abist) [reply src/sql/workbench/parts/dataExplorer/common/nodeContextUtils.ts]: https://github.com/microsoft/azuredatastudio/issues/6266

**Pull request merged**